solarforecastarbiter/io/reference_observations/common.py

```diff
@@ -415,6 +424,9 @@ def create_forecasts(api, site, variables):
         A site object.
     variables : list-like
         List of variables to make a new forecast for each of TEMPLATE_FORECASTS
+    include_probabilistic : boolean, default False
```

Comment:
this is fine. but it seems like we might want to do better at some point. maybe the arg should be `templates` instead? then the calling code decides if it wants to pass `TEMPLATE_FORECASTS` or `TEMPLATE_PROBABILISTIC_FORECASTS` or `TEMPLATE_FORECASTS + TEMPLATE_PROBABILISTIC_FORECASTS`

Reply:
looking at the `initialize_*` code below and wondering if it would be useful to define `TEMPLATE_DETERMINISTIC_FORECASTS` and `TEMPLATE_FORECASTS = TEMPLATE_DETERMINISTIC_FORECASTS + TEMPLATE_PROBABILISTIC_FORECASTS`

Reply:
That does also make it easier to reuse the code with other templates, e.g. for trials.

Reply:
documentation needs updating for new api